backend/services.endorsement-reminder/EndorsementMaintenanceService.cs

```diff
@@ -45,6 +49,14 @@ public async Task ExpireOldEndorsementRequestsAsync()
 
         foreach (var request in oldRequests)
         {
+            var email = request.Status == EndorsementRequestStatus.Approved
+                    ? request.RequestingParty!.Email!
+                    : request.RecipientEmail;
+            var token = request.Status == EndorsementRequestStatus.Created
+                    ? (Guid?)request.Token
+                    : null;
```

Comment:
When a user sends an Endorsement Request to another user, the URL contains a unique token that links the recipient to the request. The 7 day reminder email uses this token when the request is still in the status `Created`, since they haven't linked yet. The expiry email will not need this token, as the Endorsement Request is now invalid.